sklearn/tests/test_public_functions.py

```diff
@@ -230,6 +230,7 @@ def _check_function_param_validation(
     "sklearn.preprocessing.label_binarize",
     "sklearn.preprocessing.maxabs_scale",
     "sklearn.preprocessing.normalize",
+    "sklearn.preprocessing.robust_scale",
```

Comment:
```suggestion
```
Instead we can append `("sklearn.preprocessing.robust_scale", "sklearn.preprocessing.RobustScaler")` to `PARAM_VALIDATION_CLASS_WRAPPER_LIST` below.